website/docs/components/rich-tooltip/partials/code/how-to-use.md

```diff
@@ -0,0 +1,287 @@
+
+If this doesn't work for your needs/context, you can use a [generic toggle](#with-a-generic-toggle), but in this case it's up to you to make sure the implementation is compliant with the [accessibility requirements](/components/rich-tooltip?tab=accessibility).
+
+!!!
+
+The `PopoverContent` element instead is a pure container that yields the children inside the "popover" tooltip bubble. You can pass whatever content you need to it, but you are also responsible of styling and structuring it according to your needs/context.
+
+
+!!! Info
+
+Notice: we apply a CSS reset (`all: initial`) to the container, to avoid that styles applied to the parent elements leak into the tooltip content. If you find any issue with this reset please [speak with the Design System team](/about/support).
+
+!!!
+
+##### Size
+
+As you can see from the code example above, the content of the toggle doesn't get formatted by default (so that it can be used [inline with other text](#inline-with-other-text) and it inherits the typographic style from its parent element).
+
+To apply a predefined typographic styles you have to pass a `@size` argument:
+
+```handlebars
+<Hds::RichTooltip as |RT|>
+  <RT.ToggleInfoText @size="large" @icon="info" @iconPosition="trailing">More info</RT.ToggleInfoText>
+  <RT.PopoverContent>
+    <Hds::Text::Display @tag="h4" @size="200">Some title</Hds::Text::Display>
+    <Hds::Text::Body @tag="p" @size="200">Descriptive information</Hds::Text::Body>
+  </RT.PopoverContent>
+</Hds::RichTooltip>
+```
+
+In all the cases the icon size is proportional to the font size (`1em`).
+
+##### Visual organization
+
+If you need to align the toggle with other UI elements, use a parent element that wraps the content and provides the desired layout (e.g., using flexbox):
+
+```handlebars
+<div class="doc-rich-tooltip-standalone-block-flex-layout">
+  <Hds::Button @text="Your action" {{on "click" this.onClickButton}} />
+  <Hds::RichTooltip as |RT|>
+    <RT.ToggleInfoText @size="medium" @icon="info">More info</RT.ToggleInfoText>
+    <RT.PopoverContent>
+      <Hds::Text::Display @tag="h4" @size="200">Some title</Hds::Text::Display>
+      <Hds::Text::Body @tag="p" @size="200">Descriptive information</Hds::Text::Body>
+    </RT.PopoverContent>
+  </Hds::RichTooltip>
+</div>
+```
+
+#### Inline with other text
+
+If the tooltip toggle needs to be inline with other text, use the `@isInline` argument:
+
+```handlebars
+Lorem
+<Hds::RichTooltip @isInline={{true}} as |RT|>
+  <RT.ToggleInfoText>ipsum dolor</RT.ToggleInfoText>
+  <RT.PopoverContent>
+    <Hds::Text::Display @tag="h4" @size="200">Some title</Hds::Text::Display>
+    <Hds::Text::Body @tag="p" @size="200">Descriptive information</Hds::Text::Body>
+  </RT.PopoverContent>
+</Hds::RichTooltip>
+sit amet consectetur adipiscing elit.
+```
+
+In this case, is better not to assign a `@size` to the `InfoText` toggle, so that its typographic style is the same as the text that comes before/after the toggle text.
```

Comment:
```suggestion
In this case, it is better not to assign a `@size` to the `InfoText` toggle, so that its typographic style is the same as the text that comes before/after the toggle text.
```